test/utils/url.test.js

```diff
@@ -0,0 +1,72 @@
+import { appendQuery } from 'src/utils';
+import nock from 'nock';
+import expect from 'expect';
+
+/**
+ * @private
+ * Adapted from https://github.com/lakenen/node-append-query
+ */
+describe('appendQuery', function(){
+    it('should append querystring to queryless url', function () {    
+        var result = appendQuery('http://example.com/foo', 'bar=baz&beep=boop')
+            , expected = 'http://example.com/foo?bar=baz&beep=boop'
```

Comment:
This was the practice we used to share a common `client` across tests. `test/index.js` runs before any of the tests are run and is used as a setup script. At the time when the tests were being written, which was over a year ago, this was what made sense to do. Now that the SDK has matured some there might be other/better ways to manage the scope. Things like this are common in any codebase that spans over a great amount of time. Our priority has not been to rewrite the tests to since they have been working sufficiently up to this time.